### DISCUSSION TITLE
Interoperability between web3auth and torus wallet

### DISCUSSION THREAD

**Discussion by yusakapon** [Web3Auth/Web3Auth#464] [Help]
I read the [interoperability document](https://web3auth.io/docs/developer-dashboard/enable-interoperability) and checked `Allow user's private key usage in given wallets:`.  

I generated the address in my app built with web3auth.
However, when I logged into torus wallet, my address created by web3auth was not shown.

> Now, your user can use their Web3Auth private key from your application to sign transactions within Torus Wallet.

What exactly does this mean?

My expectation is that when I log into torus wallet, the address created by web3auth is linked.

Is there any way to achieve this?

I checked discussion and my question was not resolved.
https://github.com/orgs/Web3Auth/discussions/374

**Comment by shahbaz17** [comment]
Hey @yusakapon 

Did you see [this option](https://web3auth.io/docs/interoperability#flow-for-the-user-in-torus-wallet) when using https://app.tor.us?

**Comment by chaitanyapotti** [comment]
That's exactly what happens. You get multiple accounts when you login to torus wallet.
The catch is that the user has to do atleast one login into your dapp

**Comment by loydoo** [comment]
Hello!
I have the same issue: nothing is showing up despite enabling the interoperable flow option and login at least once into the dapp.
I used the quickstart from the [integration builder](https://web3auth.io/docs/integration-builder?lang=react&chain=matic&customAuthentication=no&whitelabel=no&dynamicConstructorParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3&customLogin=no#step-1), checked the Plug and play EVM based chain option (using Polygon Mumbai), logged with my Google Account and I didn't see any other address in my wallet when using https://app.tor.us/.
Any idea why?

**Comment by yashovardhan** [comment]
Hey @loydoo and @yusakapon are your apps published or you are looking at localhost applications and testing them for interoperability? Please note that localhost applications are having testnet verifiers, which means that you cannot use the interoperability while being on testnet. Once you have published your app, even on a test server somewhere, while using the mainnet or cyan verifiers, you can utlilise the interoperability flows.
